src/dataset-builder/DatasetBuilderDetails.ts

```diff
@@ -100,7 +100,7 @@ export const DatasetBuilderDetails = ({ snapshotId }: DatasetBuilderDetailsProps
             h(MarkdownViewer, [`${DATASET_NAME} dataset`]),
             div({ style: { width: '22rem', backgroundColor: 'white', padding: '1rem', marginLeft: '1rem' } }, [
               div([
-                'Use the Dataset Builder to create specific tailored data for further analyses in a Terra Workspace',
+                'Use the Data Explorer to create specific tailored data snapshots for further analyses in a Terra Workspace',
```

Comment:
The wording here seems confusing to me. Why "further"? Also why plural "analyses" vs singular "analysis"?

Reply:
I was just using the text in the mock, but I agree. I can change it to just analysis